wallet/src/Pos/Wallet/Web/Methods/Payment.hs

```diff
@@ -75,13 +77,26 @@ newPayment sa passphrase srcAccount dstAccount coin =
     -- 2. To let other things (e. g. block processing) happen if
     -- `newPayment`s are done continuously.
     notFasterThan (6 :: Second) $
-    sendMoney
+      sendMoney
         sa
         passphrase
         (AccountMoneySource srcAccount)
         (one (dstAccount, coin))
-  where
-    notFasterThan time action = fst <$> concurrently action (delay time)
+
+newPaymentBatch
+    :: MonadWalletWebMode m
+    => SendActions m
+    -> PassPhrase
+    -> NewBatchPayment
+    -> m CTx
+newPaymentBatch sa passphrase NewBatchPayment {..} = do
+    src <- decodeCTypeOrFail npbFrom
+    notFasterThan (6 :: Second) $
```

Comment:
Interestingly, this patch seems to also include changes from https://github.com/input-output-hk/cardano-sl/pull/2271. Is this expected and what we want?

Reply:
Yes, it depends on that patch and yes I think that is fine.